enhancements/authentication/automated-service-ca-rotation.md

```diff
@@ -102,29 +102,68 @@ and the functionality could be separately extracted for operator reuse.
     original expiry of the pre-rotation CA to refresh without breaking trust.
 
 - The duration of the service CA should be extended from its current value of
-  12 months to a value that exceeds the maximum expected upgrade interval
-  (currently 12 months). A value of 14 months should be sufficient.
-  - Timelines:
-    - With 12 month CA duration:
+  12 months to 26 months and the minimum CA duration should be extended to 13
+  months. These values ensure that pods will be guaranteed to be restarted in
+  a cluster supported by Red Hat before the expiry of the pre-rotation CA.
+  - The timing of upgrades is key to determining CA duration:
+    - Services that do not refresh key material automatically must be
+      restarted after CA rotation.
+    - Upgrades restart all pods, so ensuring that an upgrade takes place
+      after rotation and prior to expiry of the pre-rotation CA guarantees
+      that services will always be using valid key material.
+  - Computing CA duration:
+    - The maximum interval, *I*, between upgrades of a cluster supported by
+      Red Hat is 12 months.
+      - At most 3 OCP releases are supported at one time, and 3 releases are
+        expected in a given year.
+    - When the minimum CA duration, *M*, is reached, automatic rotation will
+      be triggered. *M* must be greater than *I* to ensure that an upgrade
+      occurs before the expiry of the pre-rotation CA.
+    - The interval between automated rotations, *R*, must also be greater
```

Comment:
where is the difference to `M`?

Reply:
Updated to specify that *R* should be greater than or equal to *M*. Given that the proposed duration is large, it makes sense to minimize *R* by simplifying to *R* = *M*.